integration-tests/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/smartcontractkit/chainlink/integration-tests
 
-go 1.18
+go 1.19
```

Comment:
Oh, whoops. I had bumped this also in my other PR at some point. Must have lost it in a rebase :thinking: